scripts/test.js

```diff
@@ -93,12 +70,17 @@ function test(rebuild, browsersArg) {
 function main() {
     const arguments = process.argv.slice(2);
     const rebuild = (arguments.indexOf('--rebuild') > -1);
+    const linkOptionIdx = arguments.indexOf('--link');
+    let link = true;
+    if (linkOptionIdx > -1) {
+        link = arguments[linkOptionIdx+1];
+    }

```

Comment:
Why you need to check the value of the `--link` argument? Can you just see if the argument exists or not like the `--rebuild` argument above?
Then the code will become:
`const link = (arguments.indexOf('--link') > -1);`

Reply:
Because the default value is true and we turn it off using  --link false.
Reversed the logic and added --no-link instead.